job_definitions/update_index.yml

```diff
@@ -0,0 +1,55 @@
+{% for environment in ['preview', 'staging', 'production'] %}
+- job:
+    name: "update-index-{{ environment }}"
+    display-name: "Update index - {{ environment }}"
+    project-type: freestyle
+    description: >
+      This job updates an existing ElasticSearch index with services or briefs as they exist in the database
+
+      In all likelihood you want to reindex all existing briefs or services to their corresponding index.
+
+      Visiting either https://search-api.<STAGE>.marketplace.team/_status or
+      https://search-api.digitalmarketplace.service.gov.uk/_status will show you currently available indexes and their
+      aliases.
+
+      To reindex all briefs your object will be 'briefs', the index will be 'briefs-digital-outcomes-and-specialists'
+      and the frameworks should be 'digital-outcomes-and-specialists,digital-outcomes-and-specialists-2'
+
+      To reindex the g-cloud services list the object is 'services', the index is 'g-cloud-10' and the frameworks would
+      be 'g-cloud-10'
+
+      However you may be testing a new index and want to input its name and frameworks manually.
+    parameters:
+      - choice:
+          name: OBJECTS
+          choices:
+            - briefs
+            - services
+          default: services
+      - string:
+          name: INDEX
+          default: {{ search_config['services'][environment].default_index }}
+          description: "The current name or alias of the index you want to update (briefs-digital-outcomes-and-specialists or g-cloud-10)"
+      - string:
+          name: FRAMEWORKS
+          description: >
+            Comma-separated list of framework slugs that should be indexed
+            (eg 'g-cloud-10' or 'digital-outcomes-and-specialists,digital-outcomes-and-specialists-2').
+    triggers:
+      - timed: "H 2 * * *"
```

Comment:
As above.

Reply:
This one should be timed right?

Reply:
No, I don't think so. I see these two jobs as things that need to be wrapped and customised depending on if we're indexing briefs or services. I think the wrappers should be timed.

Reply:
Ah I think I understand now